castervoice/lib/settings.py

```diff
@@ -301,7 +312,7 @@ def _get_defaults():
 
             # EXECUTABLES
             "AHK_PATH":
-                str(Path(_BASE_PATH).joinpath(ahk_path_default)),
+                str(Path(ahk_path_default)),
```

Comment:
This one I'm not so sure of. I don't think that this was ever right

Reply:
Yes as `ahk_path_default` should already include the base path